packages/marshal/src/encodePassable.js

```diff
@@ -182,37 +182,38 @@ const encodeBigInt = n => {
   }
 };
 
+const rBigIntPayload = /([0-9]+)(:([0-9]+$|)|)/s;
```

Comment:
Good. Consolidating the regexp makes this much more declarative and readable. Nice!